test/methods.jl

```diff
@@ -126,6 +126,32 @@ end
     @test_throws ArgumentError combine()
 end
 
+@testset "dark subtraction" begin
+    # testing non-mutating version
+    @test @inferred(subtract_dark(ones(5, 5), ones(5, 5))) == zeros(5, 5)
+    @test @inferred(subtract_dark(ones(5, 5), ones(5, 5), data_exposure = 1, dark_exposure = 4)) == fill(0.75, 5, 5)
+    @test @inferred(subtract_dark(ones(5, 5), ones(5, 5), data_exposure = 13, dark_exposure = 17)) ≈ fill(Float32(4/17), 5, 5)
```

Comment:
I'm not sure how this one is useful for more general testing type-stability, since the arguments of `subtract_dark` are `Array{Float64}`

Reply:
I missed `Float32` over there. :sweat_smile: